vendor/github.com/gocarina/gocsv/README.md

```diff
@@ -0,0 +1,151 @@
+Go CSV
```

Comment:
So far we added a vendor directory in the module directly if the dependency was specific to the module. TBH with csv it is fine from my point of view to have it in the top dependencies as this will be also used in other modules.